### PR TITLE
Improve first-run registration

### PR DESCRIPTION
**Pull request by hitesh96db**
Addresses all the issues pointed out here https://github.com/privly/privly-applications/pull/180 except for the integration tests.

### CODE REVIEW

File: test/run_each.sh [hunk 1]
```diff
@@ -34,7 +34,7 @@ runTest () {
 runTest 'Help/test/*.js,Help/js/*.js'
 runTest 'History/js/*.js,History/test/*.js'
 runTest 'Login/js/*.js,Login/test/*.js'
-runTest 'Pages/js/options.js,Pages/js/tests/*.js'
+runTest 'Pages/js/*.js,Pages/js/tests/*.js'
```
Comment (smcgregor) [inline test/run_each.sh]: This will test both the options page and the first run page. I think they likely need to be tested separately at the moment.

**Comment by irdan** [comment]
@hitesh96db Sorry for the delay in reviewing.  Looking at this now.

**Comment by irdan** [comment]
I know this wasn't mentioned in the original issue, but making a similar modal dialogue appear when a user clicks `sign-in` would go a long way towards making the page more usable.  Right now the redirects make it difficult for a user to get back to the first run page.

While the change isn't required in this PR, it must be in place before landing in a release.  Let me know if you want to implement it in this PR or not.

**Comment by irdan** [comment]
Excellent tests!  Overall this looks great.

I think there are definitely some usability issues that this new functionality presents.  For example, in the message telling the user they will receive an email if they are not already in our database we should provide a 'help' link in case they run in to problems.  I believe that type of feature is outside of the scope of the issue being addressed here, but an issue should be created detailing the need for it.

**Comment by hitesh96db** [comment]
> Right now the redirects make it difficult for a user to get back to the first run page.

I've added a "_blank" target to the "Sign in" link, which would make the page open in a new tab, and should help the user to still get back to the first-run page. I think it's best to keep the "Sign In" page separate.

> of the issue being addressed here, but an issue should be created detailing the need for it.

Will do :)

**Comment by hitesh96db** [comment]
@irdan Removed extra whitespaces. I'll create another PR to address the issue you mentioned here.

**Comment by smcgregor** [comment]
@irdan @hitesh96db What is the status on this? I am trying to clean up pull requests.

**Comment by hitesh96db** [comment]
This PR is ready for review.

**Comment by irdan** [comment]
@hitesh96db Sorry for the delay.  I am going to merge this.  Keep in mind that it is a lot easier for me to review if you wait to rebase until right before merging.

**Pull request merged**